Dispatcher/UI/CustomButton.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  CustomButton.swift
+//  Dispatcher
+//
+//  Created by Hadar Basson on 28/08/2023.
+//
+
+import UIKit
+
+class CustomButton: UIButton {
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupButton()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setupButton()
+    }
+    
+    private func setupButton() {
+        let widthConstraint = widthAnchor.constraint(equalToConstant: 335)
+        let heightConstraint = heightAnchor.constraint(equalToConstant: 36)
+        
+        widthConstraint.isActive = true
```

Comment:
if we are not using .activate function for constraint 
we are using 
widthAnchor.constraint(equalToConstant: 335).isActive = true